src/js/components/Select/doc.js

```diff
@@ -26,6 +26,27 @@ export const doc = Select => {
     closeOnChange: PropTypes.bool
       .description('Wether to close the drop when a selection is made.')
       .defaultValue(true),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom'])
+          .description(
+            `Add a clear value to the top or at the bottom of the
+            container. By default no clear option is present.`,
+          )
+          .defaultValue('top'),
+        label: PropTypes.string
+          .description('Label for the clear selection item')
+          .defaultValue('Clear selection'),
```

Comment:
@ericsoderberghp just confirming this approach is compatible with compliancy to I18N, the user is able to send the translated string directly to the label prop.

Reply:
@oorestisime I'm thinking we should move this message to the `messages` prop for better I18N support, thoughts?

Reply:
I think, as long as this is a string it will work with i18n.